src/libraries/System.Security.Cryptography/tests/HmacMD5Tests.cs

```diff
@@ -10,11 +10,12 @@
 namespace System.Security.Cryptography.Tests
 {
     [SkipOnPlatform(TestPlatforms.Browser, "Not supported on Browser")]
+    [ConditionalClass(typeof(HmacMD5Tests.Traits), nameof(HmacMD5Tests.Traits.IsSupported))]
     public class HmacMD5Tests : Rfc2202HmacTests<HmacMD5Tests.Traits>
     {
         public sealed class Traits : IHmacTrait
         {
-            public static bool IsSupported => true;
+            public static bool IsSupported => !PlatformDetection.IsAzureLinux;
```

Comment:
In HKDF these two skip clauses got combined.  Should they here, as well?

(Remove SkipOnPlatform, make IsSupported be `!IsAzureLinux && !IsBrowser`)

Reply:
Yeah, that's a good idea. Done.